src/classes/HMSSDK.tsx

```diff
@@ -104,10 +106,16 @@ export class HMSSDK {
     appGroup?: String;
     preferredExtension?: String;
   }) {
+    const { version } = require('../../package.json');
```

Comment:
@samarthjain10 Will this path `../../package.json` work for client apps as well?